.github/workflows/integration_test.yaml

```diff
@@ -7,9 +7,10 @@ jobs:
   integration-tests:
     uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@add_support_for_canonical_k8s
     secrets: inherit
-    runs-on: ['self-hosted', 'x64', 'noble-large']
     with:
       provider: k8s
       use-canonical-k8s: true
+      self-hosted-runner: true
+      self-hosted-runner-label: "large"
```

Comment:
Are you sure you need large?

Reply:
I can't bootstrap canonical k8s with edge runners due to them not having enough disk space (maybe canonical k8s controllers have a higher resource requirements than microk8s ones). So I skipped medium since I'd add more tests in the future.